### DISCUSSION TITLE
Where is the Analyze package?

### DISCUSSION THREAD

**Discussion by TechAkayy** [vuedx/languagetools#259] [Q&A] [answered]
Hi there, 

I'm after the Analyze package, Rahul mentioned it here (https://youtu.be/dh7Mj0cCVjY?list=PLMapeSQlw8ybn7CqkqsII7f_HwKN-k-SM&t=1030). I understand many packages have gone through minor renaming. But, I see the Analyze package going missing... 

Description of it was that its an SFC inspector - A static component API extractor for Vue. Can someone advise where I can find it if it was moved/renamed. Thanks bunch! 

Here is how it looked like a while ago - https://github.com/vuedx/languagetools/tree/b097e7e7258499beb2b1bcf9fad55a1eedd66d2f/packages/analyze

**Comment by znck** [comment, marked as answer]
We changed the core of VueDX, now analyze package is not used so it didn't survive the rewrite. You can still find it on [v0 branch](https://github.com/vuedx/languagetools/tree/v0/packages/analyze). We may bring it back in future for [https://github.com/vuedx/preview].

**Comment by TechAkayy** [comment]
Thanks bunch @znck for the reference to v0 branch. There is a lack of a library like the erstwhile 'analyze' package which can help us as a static component API extractor. 

I have done many searches and the closest I have got are the Vue2 ones (not maintained actively) like https://gitlab.com/vuedoc/parser &  the parser of https://vue-styleguidist.github.io/docs/Docgen.html 

I will try to use the v0 branch analyze package. But, I still see a lot of value in maintaining something similar that helps us as a SFC static extractor of props & other info. Not sure if the boat has already sailed with the rewrite though, still hopeful to see it in **Preview** as you mentioned. Also, please let me know if you knew & could suggest an alternative. 

Thanks again, I'm a big fan of your minion world helping with all the Vue ecosystem tooling!
